mod/Jailbreak.LastRequest/LastRequestManager.cs

```diff
@@ -153,11 +153,23 @@ public HookResult OnPlayerDeath(EventPlayerDeath @event, GameEventInfo info)
     public HookResult OnPlayerDisconnect(EventPlayerDisconnect @event, GameEventInfo info)
     {
         var player = @event.Userid;
+
+        if (player == null) return HookResult.Continue;
+        
         if (!player.IsReal() || ServerExtensions.GetGameRules().WarmupPeriod)
             return HookResult.Continue;
+        
         if (IsLREnabled)
             return HookResult.Continue;
 
+        var activeLr = ((ILastRequestManager)this).GetActiveLR(player);
+
+        if (activeLr != null || activeLr.state != LRState.Active)
```

Comment:
This should probably be &&, as otherwise activeLr would have to be null for this check to be done.